menu-animation/menu.css

```diff
@@ -0,0 +1,118 @@
+.menu {
+  --ripple-animation-time: 0.4s;
+  --animation-delay: calc(var(--ripple-animation-time) / 2);
+  --animation-time: 0.6s;
+  --transition-fn: cubic-bezier(0.39, 0, 0.7, 0.39);
+  --main-color: #fff;
+
+  display: flex;
+  align-items: center;
+  height: 130px;
+  padding: 10px;
+  overflow: hidden;
+  border-radius: 0 0 20px 20px;
+  background-color: var(--main-color);
+}
+
+.menu-item {
```

Comment:
So far this looks nice! But is it functional? Here's 2 things missing to make this ready for real-world :)
- I noticed there isn't any `:hover` styling, could you do that? Perhaps only changing the color on interaction.
- Have you tried to interact with these buttons in any other way than the mouse? Maybe using keyboard (Tab + Enter)

Reply:
Yes. I should think about it) Will be fixed!